index.html

```diff
@@ -40,13 +40,274 @@ <h3 class="large-text"> I can help you build a product, feature or website. Look
+                </div>
+                <div class="language-box" id="js">
+                  <h4 class="language-text">JavScript</h4>
+                </div>
+                <div class="language-box" id="html">
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
+              <br>
+              <div class="see-container">
+                <a class="see-project">See Project</a>
+              </div>
+            </div>
+          </div>
+          <br>   
+          <div class="card">
+            <div class="cardlet">
+              <h3 class="project-title">Multi-Post Stories<br>Gain+Glory</h3>
+              <div class="languages-container">
+                <div class="language-box" id="ruby">
+                  <h4 class="language-text">Ruby on rails</h4>
+                </div>
+                <div class="language-box" id="css">
+                  <h4 class="language-text">css</h4>
+                </div>
+                <div class="language-box" id="js">
+                  <h4 class="language-text">JavScript</h4>
+                </div>
+                <div class="language-box" id="html">
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
+              <br>
+              <div class="see-container">
+                <a class="see-project">See Project</a>
+              </div>
+            </div>
+          </div>
+          <br>        
+          <div class="card">
+            <div class="cardlet">
+              <h3 class="project-title">Multi-Post Stories<br>Gain+Glory</h3>
+              <div class="languages-container">
+                <div class="language-box" id="ruby">
+                  <h4 class="language-text">Ruby on rails</h4>
+                </div>
+                <div class="language-box" id="css">
+                  <h4 class="language-text">css</h4>
+                </div>
+                <div class="language-box" id="js">
+                  <h4 class="language-text">JavScript</h4>
+                </div>
+                <div class="language-box" id="html">
+                  <h4 class="language-text">html</h4>
+                </div>
+              </div>
+              <br>
+              <div class="see-container">
+                <a class="see-project">See Project</a>
+              </div>
+            </div>
+          </div>
+          <br>
+        </div>
+      </section>
+      <br>
+      <img src="images/illustration 1- about me-mobile@2x.png" id="paper-bird-1" alt="decorative image">
+      <br>
+      <section class="about-myself">
+        <h2 class="about-me">About Me</h2>
+        <div class="large-text-container"></div>
+        <h3 class="large-text-2"> I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hesitate to contact me.</h3>
```

Comment:
- [ ] I would suggest using a `<p>` tag instead of the `<h3>` for this text since this is not considered a title or header :smiley: